frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -343,7 +343,12 @@ const HostDetailsPage = ({
     const hostEnrolled = ["On (automatic)", "On (manual)"].includes(
       host?.mdm.enrollment_status ?? ""
     );
-    return userHasPermission && hostEnrolled;
+    return (
+      userHasPermission &&
+      hostEnrolled &&
+      config?.mdm.enabled_and_configured &&
+      mdm?.name === "Fleet"
```

Comment:
I think we should either have a typescript `enum` somewhere or do this in `frontend/interfaces/mdm.ts`:

```ts
export interface IMdmSolution {
  id: number;
  /* This list must be manually kept in sync with the list of known MDM
   * solutions in server/fleet/hosts.go
   * TODO: automate type generation
   */
  name:
    | "Kandji"
    | "Jamf"
    | "VMware Workspace ONE"
    | "Intune"
    | "SimpleMDM"
    | "Fleet"
    | null;
  server_url: string;
  hosts_count: number;
}
```

Whatever works depending on the patterns you folks are using in the UI.

Reply:
Im not sure this offers us much and adds the chore of manually updating this property as more MDMs are added (which I can see us easily forgetting to do). I do not think we are checking the mdm name anywhere else in the codebase either, so not sure we need to change `name` to a union of exact string names.

Reply:
Up to you folks! I'm not familiar enough with our UI patterns/standards.

What I can tell you is that the list of names is a pretty well defined list! so, the benefit of defining this in the types is that it prevents typos or wrong values.

Reply:
I also just realized that this branch is based of a release branch, I'm guessing we're not going to cherry-pick the commit to `main`, so it's probably not worth doing here!